string object_name
geometry_msgs/PoseStamped pose
geometry_msgs/Point bb_lwh
---
bool completed
string msg